templates/commands/render/templatesource/source.go

```diff
@@ -86,17 +87,17 @@ var realSourceParsers = []sourceParser{
 	},
 }
 
-// ParseSource maps the input template source to a particular kind of source
-// (e.g. git) and returns a downloader that will download that source.
+// ParseSourceWithWorkingDir maps the input template source to a particular kind
```

Comment:
Can this be unexported (lower case)?

Maybe also consider "CWD" or "WD" (in the vein of os.Getwd()) instead of "WorkingDir", it's quite a long function name.

Reply:
`parseSourceWithCwd` it is!